DeepLinkKit/Categories/NSString+DPLQuery.h

```diff
@@ -42,4 +42,32 @@
  */
 - (NSString *)DPL_stringByReplacingPercentEscapesUsingEncoding:(NSStringEncoding)encoding;
 
+
+
+///---------------------
+/// @name Array Literals
+///---------------------
+
+
+/**
+ Checks if string ends with array literal ('[]')
+ @return YES if string ends with '[]', NO otherwise
+ */
+- (BOOL)DPL_containsArrayLiteral;
+
+
+/**
+ Checks if string contains key with array literal
+ @return YES if string contains key with '[]' attached (e.g. "key[]"),
+          NO otherwise (e.g. "key")
+ */
+- (BOOL)DPL_containsArrayLiteralWithKey:(NSString *)key;
+
+
+/**
+ Removes array literal ('[]') from the string
+ @return String without array literal ('[]')
+ */
+- (NSString *)DPL_removeArrayLiteral;
```

Comment:
Thanks for noticing, will correct it shortly.

On Thursday, April 7, 2016, Chris Maddern <notifications@github.com> wrote:

> In DeepLinkKit/Categories/NSString+DPLQuery.h
> <https://github.com/button/DeepLinkKit/pull/98#discussion_r58930895>:
>
> > +- (BOOL)DPL_containsArrayLiteral;
> > +
> > +
> > +/**
> > + Checks if string contains key with array literal
> > + @return YES if string contains key with '[]' attached (e.g. "key[]"),
> > +          NO otherwise (e.g. "key")
> > + */
> > +- (BOOL)DPL_containsArrayLiteralWithKey:(NSString *)key;
> > +
> > +
> > +/**
> > + Removes array literal ('[]') from the string
> > + @return String without array literal ('[]')
> > + */
> > +- (NSString *)DPL_removeArrayLiteral;
>
> Convention would indicate:
> DPL_stringByRemovingArrayLiteral or something similar
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/button/DeepLinkKit/pull/98/files/7cebc8717560a0a64fb675df53f041ef2e2d7acb#r58930895>
>


-- 
Best regards,
Pavel Pantus